app/Http/Controllers/Admin/ProjectController.php

```diff
@@ -51,12 +52,16 @@ public function index(
     ) {
         $projects = $this->repository->getAll();
 
+        // Getting all users
+        $users = User::where('is_admin', 0)->get();
```

Comment:
This should probably be changed a method on the user repository and inject the repository in the method